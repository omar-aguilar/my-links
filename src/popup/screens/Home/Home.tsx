import { FunctionComponent } from 'react';
import useDocumentTitle from '../../hooks/useDocumentTitle';

const Home: FunctionComponent = () => {
  useDocumentTitle('My Links - Home');
  return (
    <>
      <div>My Links Home</div>;
    </>
  );
};

export default Home;
