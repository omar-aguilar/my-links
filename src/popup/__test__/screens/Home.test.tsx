import { render } from '@testing-library/react';
import Home from '../../screens/Home';

describe('Home', () => {
  it('renders without exploding', () => {
    const { container } = render(<Home />);
    expect(container.ownerDocument.title).toEqual('My Links - Home');
    expect(container.firstChild).toMatchSnapshot();
  });
});
