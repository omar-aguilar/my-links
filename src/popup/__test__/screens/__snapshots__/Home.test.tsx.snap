// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without exploding 1`] = `
<div>
  My Links Home
</div>
`;
